content/en/real_user_monitoring/session_replay/mobile/privacy_options.md

```diff
@@ -410,9 +408,148 @@ With the `allow` setting enabled, all text is revealed.
 {{% /tab %}}
 {{< /tabs >}}
 
+## Privacy Overrides
+
+The sections above describe the global masking levels that apply to the entire application. However, it is also possible to override these settings at the view level. The same privacy levels as above are available for text and inputs, images, touches, and an additional setting to completely hide a specific view.
+
+To ensure overrides are recognized properly, they should be applied as early as possible in the view lifecycle. This prevents scenarios where Session Replay might process a view before applying the overrides.
+
+Privacy overrides affect views and their descendants. This means that even if an override is applied to a view where it might have no immediate effect (e.g., applying an image override to a text input), the override still applies to all child views. 
```

Comment:
```suggestion
Privacy overrides affect views and their descendants. This means that even if an override is applied to a view where it might have no immediate effect (for example, applying an image override to a text input), the override still applies to all child views. 
```